sdk/face/ai-vision-face-rest/src/responses.ts

```diff
@@ -836,1001 +667,1211 @@ export interface GetLargeFaceListFacesDefaultResponse extends HttpResponse {
+  body: Array<LivenessSessionItemOutput>;
 }
 
-export interface AddPersonGroupPersonFaceFromUrlDefaultHeaders {
+export interface GetLivenessWithVerifySessionsDefaultHeaders {
   /** String error code indicating what went wrong. */
   "x-ms-error-code"?: string;
 }
 
-export interface AddPersonGroupPersonFaceFromUrlDefaultResponse extends HttpResponse {
+export interface GetLivenessWithVerifySessionsDefaultResponse extends HttpResponse {
   status: string;
   body: FaceErrorResponseOutput;
-  headers: RawHttpHeaders & AddPersonGroupPersonFaceFromUrlDefaultHeaders;
+  headers: RawHttpHeaders & GetLivenessWithVerifySessionsDefaultHeaders;
 }
 
-/** A successful call returns a new persistedFaceId. */
-export interface AddPersonGroupPersonFace200Response extends HttpResponse {
+/** The request has succeeded. */
+export interface GetLivenessWithVerifySessionAuditEntries200Response extends HttpResponse {
   status: "200";
-  body: AddFaceResultOutput;
+  body: Array<LivenessSessionAuditEntryOutput>;
 }
 
-export interface AddPersonGroupPersonFaceDefaultHeaders {
+export interface GetLivenessWithVerifySessionAuditEntriesDefaultHeaders {
   /** String error code indicating what went wrong. */
   "x-ms-error-code"?: string;
 }
 
-export interface AddPersonGroupPersonFaceDefaultResponse extends HttpResponse {
+export interface GetLivenessWithVerifySessionAuditEntriesDefaultResponse extends HttpResponse {
   status: string;
   body: FaceErrorResponseOutput;
-  headers: RawHttpHeaders & AddPersonGroupPersonFaceDefaultHeaders;
+  headers: RawHttpHeaders & GetLivenessWithVerifySessionAuditEntriesDefaultHeaders;
+}
+
+export interface GetSessionImage200Headers {
+  /** The format of the HTTP payload. */
+  "content-type": "application/octet-stream";
 }
 
 /** The request has succeeded. */
-export interface DeletePersonGroupPersonFace200Response extends HttpResponse {
+export interface GetSessionImage200Response extends HttpResponse {
   status: "200";
+  /** Value may contain any sequence of octets */
+  body: Uint8Array;
+  headers: RawHttpHeaders & GetSessionImage200Headers;
 }
 
-export interface DeletePersonGroupPersonFaceDefaultHeaders {
+export interface GetSessionImageDefaultHeaders {
   /** String error code indicating what went wrong. */
   "x-ms-error-code"?: string;
 }
 
-export interface DeletePersonGroupPersonFaceDefaultResponse extends HttpResponse {
+export interface GetSessionImageDefaultResponse extends HttpResponse {
   status: string;
   body: FaceErrorResponseOutput;
-  headers: RawHttpHeaders & DeletePersonGroupPersonFaceDefaultHeaders;
+  headers: RawHttpHeaders & GetSessionImageDefaultHeaders;
 }
 
-/** A successful call returns target persisted face's information (persistedFaceId and userData). */
-export interface GetPersonGroupPersonFace200Response extends HttpResponse {
-  status: "200";
-  body: PersonGroupPersonFaceOutput;
+export interface CreatePerson202Headers {
+  "operation-location": string;
+  location: string;
```

Comment:
are they both headers for polling url ? is it a duplication here?

Reply:
They are different:
- Operation-Location: it's the polling URL.
- Location: it's the GET URL of the created person, i.e., https://learn.microsoft.com/en-us/rest/api/face/person-directory-operations/get-person?view=rest-face-v1.2-preview.1&tabs=HTTP